src/main/java/seedu/address/MainApp.java

```diff
@@ -93,23 +97,20 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
             initialAddressData = new AddressBook();
         }
 
-        /* Once done with the storage, you can uncomment this and connect this properly
         try {
-            eventBookOptional = storage.readeventBook(); //Implement this
+            eventBookOptional = storage.readEventBook(); //Implement this
```

Comment:
You might want to consider removing the comment since you're done implementing it